sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamValuesRelTest.java

```diff
@@ -91,4 +91,17 @@ public void testValues_onlySelect() throws Exception {
                 .getRows());
     pipeline.run();
   }
+
+  @Test
+  public void testValues_selectEmpty() throws Exception {
+    String sql = "select 1, '1' FROM string_table WHERE false";
```

Comment:
I think it's better to assign aliases: `SELECT 1 AS x, '1' AS y ...` so that in unlikely case Calcite changes the convention of calling these `EXPR$..` the test doesn't break

Reply:
Calcite's rules might lose alias so alias is not 100% stable.

Reply:
I think we should disable this behavior. If a user has a sink after `SqlTransform` they would probably expect the fields in the sink to match the fields in the query.

Reply:
Agreed. `CalcRemoveRule` is disabled already. Not sure if there is other case.

We need a set of tests to test logical plans. It could be done along with the work of enabling all useful optimization rules.